item.json

```diff
@@ -0,0 +1,126 @@
+{
```

Comment:
This file should not live at the top-level of the repository ... is it meant to be an example?

Reply:
Not sure how it ended up at the top-level. There was already an item file in the examples folder. I removed the "item.json" from the top-level.